api/basic/add-pin.ts

```diff
@@ -125,7 +127,7 @@ export async function putIfNotExists ({ cid, dynamo, table }: UpsertPinInput): P
     }))
     logger.info({ code: 'DYNAMO_PUT' }, 'New pin saved')
     // Pin was saved, so return it
-    return pin
+    return { shouldQueue: true, pin }
```

Comment:
I guess it's something not related to this PR, anyway `shouldQueue` is not related to the function context

Reply:
yes, this is more a quick way to do it. It requires a better refactoring